app/main/plugins/core/basic-apps/linux.js

```diff
@@ -0,0 +1,58 @@
+import { remote } from 'electron'
+import path from 'path'
+import fs from 'fs'
+import uniq from 'lodash/uniq'
+import { shellCommand } from 'cerebro-tools'
+
+let appDirs = [
+  path.join(remote.app.getPath('home'), '.local', 'share')
+]
+
+if (!!process.env.XDG_DATA_DIRS) {
+  appDirs = [
+    ...appDirs,
+    ...process.env.XDG_DATA_DIRS.split(':')
+  ]
+}
+
+appDirs = appDirs.map(dir => path.join(dir, 'applications'))
+
+export const DIRECTORIES = uniq([
+  ...appDirs,
+  path.join('usr', 'share', 'app-install', 'desktop')
```

Comment:
@KELiON we also need `/usr/share/applications` and `/usr/local/share/applications` here